application/modules/g/controllers/ContentController.php

```diff
@@ -54,6 +54,21 @@ public function indexAction() {
     }
 
     /**
+     * Reset OpCache
+     *
+     * @return Void
+     */
+    public function opcacheresetAction() {
+        $this->_helper->viewRenderer->setNoRender(true);
+        if (function_exists('opcache_reset')) {
+            $result = opcache_reset();
+            echo $result ? 'opcache resetted' : 'opcache reset failed';
+            return;
+        }
+        echo 'no opcache installed';
```

Comment:
Wouldn't it be better to just throw an exception instead of echoing? 🤔

Reply:
Returning a function with a default output (ask Harmen for a good example) might also be good. Just echo'ing is not so nice, since we're pretty much always reading this from CLI...

Reply:
I was wrong, this specific part is always triggered from an HTTP context. So disregard my above comment.